xml/System.Text/Encoding.xml

```diff
@@ -2647,18 +2655,26 @@
   
- In addition to the encodings that are intrinsically supported on a specific platform version of the .NET Framework, the <xref:System.Text.Encoding.GetEncoding%28System.Int32%29> method returns any additional encodings that are made available by registering an <xref:System.Text.EncodingProvider> object.  
-  
- To get the encoding associated with the default ANSI code page in the operating system's regional and language settings, you can either supply a value 0 for the `codepage` argument or, on the full .NET Framework on the Windows desktop, retrieve the value of the <xref:System.Text.Encoding.Default%2A> property. To determine the default code pages used on the system, use the Windows [GetSystemDefaultLangID](http://msdn.microsoft.com/library/windows/desktop/dd318120.aspx) function. To determine the current ANSI code page, call the Windows [GetACP](http://msdn.microsoft.com/library/windows/desktop/dd318070.aspx) function from the full .NET Framework on the Windows desktop.  
+ In addition to the encodings that are intrinsically supported on a specific platform version of the .NET Framework, the <xref:System.Text.Encoding.GetEncoding%2A> method returns any additional encodings that are made available by registering an <xref:System.Text.EncodingProvider> object. If the same encoding has been registered by multiple <xref:System.Text.EncodingProvider> objects, this method returns the last one registered.  
+
+You can also supply a value of 0 for the `codepage` argument. Its precise behavior depends on whether any encodings have been made available by registering an <xref:System.Text.EncodingProvider> object:
+
+- If one or more encoding providers have been registered, it returns the encoding of the last registered provider that has chosen to return a encoding when the <xref:System.Text.Encoding.GetEncoding%2A> method is passed a `codepage` argument of 0.     
+
+- On the .NET Framework, if no encoding provider has been registered, if the <xref:System.Text.Encoding.CodePagesEncodingProvider> is the registered encoding provider, or if no registered encoding provider handles a `codepage` value of 0, it returns the default ANSI code page in the operating system's regional and language settings.
+
+- On .NET Core, if no no encoding provider has been registered or if no registered encoding provider handles a `codepage` value of 0, it returns the <xref:System.Text.UTF8> encoding.
+
+To determine the default code pages used on Windows systems, call the Windows [GetSystemDefaultLangID](http://msdn.microsoft.com/library/windows/desktop/dd318120.aspx) function from the full .NET Framework on the WIndows desktop. To determine the current ANSI code page on Windows systems, call the Windows [GetACP](http://msdn.microsoft.com/library/windows/desktop/dd318070.aspx) function from the full .NET Framework on the Windows desktop.  
```

Comment:
>call the Windows [GetSystemDefaultLangID](http://msdn.microsoft.com/library/windows/desktop/dd318120.aspx) [](start = 61, length = 106)

This is wrong. default lang id is different than ACP. You need to delete this part of the paragraph and keep only the part for GetACP